### DISCUSSION TITLE
Improve the guidance for providing system/environment details

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#7096] [Discussion]
Currently, we ask for this in the "Bug" issue form:
![Issue form 1](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/e2334b54-e7f2-46c0-ae8c-fd8d7a9100cc)
But even with that, I still see people not filling in this information. Also, it's not clear how you provide a "hash".

If anyone has any ideas on how to improve this experience, let me know. @WorldLanguages @Hans5958, any comment?

I'll make my own suggestions below, too.

**Comment by DNin01** [comment]
One option I thought of is removing the \ so that it's more obvious that the fields are blank, maybe? Careful, though - we still want people to know to specify the browser/OS version. It might also help to separate the Scratch Addons version line into its own field that is required so users don't just skip providing any information (having a placeholder value counts as a required form being filled in).

![Issue form 2](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/51f12a62-637b-4bc7-84c0-a87594a13946)

**Reply by WorldLanguages** [reply]
Making it mandatory to type in the version could lead to less bug reports (surely it will not bring more bug reports) so I'm not convinced
Most bug reports are valid and happen on the latest version anyway

**Comment by DNin01** [comment]
We could explain information in the input/textarea descriptions, but it could be easily missed as it's small.

And if we do that, should we link to the contributing guidelines for additional info or our website?

**Comment by Hans5958** [comment]
An ideal way to do this is to have a link to the issue template that has been prefilled with enviroment info inside SA, but I don't think this is feasable.

I think it's an issue of people being lazy. Those who don't care enough to add that info wouldn't care less if we change things up.

PS. I'm also thinking to use the website to bridge the useragent, but you would need more links for each of the template, which means it wouldn't be nice.

**Reply by DNin01** [reply]
> have a link to the issue template that has been prefilled with enviroment info inside SA

You can do that? How?

That would be really cool, we have to consider adding a link to the bug report form from SA or from somewhere on our website like the feedback form.

Related: #6451

**Reply by Hans5958** [reply]
I saw that editing on https://iptv-org.github.io would mean creating an issue with a template, with some fields filled. Pick a channel and try to edit it.

For example, here's the URL when I try to edit a channel.

```
https://github.com/iptv-org/database/issues/new?labels=channels%3Aedit&template=__channels_edit.yml&title=Edit%3A%20TVRI%20Nasional&id=TVRINasional.id
```
